bin/sdr

```diff
@@ -88,10 +103,17 @@ class CLI < Thor
          banner: 'ENVIRONMENT',
          desc: "Environment (#{Settings.supported_envs.keys.map(&:to_s)})",
          aliases: '-e'
+  option :control_master,
+         type: :boolean,
+         default: false,
+         desc: 'Use control master for SSH connections',
+         aliases: '-c'
```

Comment:
```suggestion
  option :skip_control_master,
         type: :boolean,
         default: false,
         desc: 'Skip checking for active SSH control master connection'
```

Reply:
That makes sense to me. I'll update this and the readme.